docs/guides/mozilla-sops.md

```diff
@@ -131,10 +133,27 @@ AWS IAM Role example:
 }
 ```
 
+#### Azure
+
 When using Azure Key Vault you need to authenticate the kustomize controller either by passing
 [Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault)
 or with [add-pod-identity](https://github.com/Azure/aad-pod-identity).
 
+#### Google Cloud
+
+Please ensure that the GKE cluster has Workload Identity enabled.
+
+1. Create a service account with the role `Cloud KMS CryptoKey Encrypter/Decrypter`.
+2. Create an IAM policy binding between the GCP service account to the `default` service account of the `flux-system`.
+3. Annotate the `default` service account in the `flux-system` with the GCP service account.
+
+```sh
+kubectl annotate serviceaccount \
```

Comment:
Is it wise to use a `kubectl annotate` example when we are trying to make people do GitOps?

Reply:
Yes this is ok, the default SA is generated by Kubernetes, when we'll switch to dedicated SAs this needs to be included in the bootstrap procedure.